packages/authentication/package.json

```diff
@@ -4,9 +4,10 @@
   "description": "A LoopBack component for authentication support.",
   "scripts": {
     "acceptance": "mocha --opts ../../test/mocha.opts 'test/acceptance/**/*.ts'",
-    "build": "npm run build:lib && npm run build:lib6",
+    "build": "npm run build:lib && npm run build:lib6 && npm run build:apidocs",
```

Comment:
Also one another reason, I wanted apidocs to use strongdocs, all that it has to offer, meaning parsing as well as complete rendering. Right now apidocs is using the strong-docs for parsing, and half of rendering, and adds a lot of rendering itself. My approach compeltely uses what strong-docs has to offer, including the complete rendering. apidocs is adding to generated documentation only the stuff that doesn't come from strong-docs, namely the versions part on the left, the modules part of the right, and the branding (title).